src/main/resources/db/migration/V1_99__activity_migration.sql

```diff
@@ -0,0 +1 @@
+alter table activity_migration_mapping alter column activity_id drop not null;
```

Comment:
We need to end NOMIS activities even if they weren't migrated. To achieve this we'll record them in the mapping table with a null activity_id. This tells us that they weren't migrated but we know which NOMIS activity to end.